src/shaders/metallic-roughness.frag

```diff
@@ -523,17 +619,18 @@ void main()
     color += emissive;
 #endif
 
+    // TODO: Remove this? Doesn't work with multiple light sources.
```

Comment:
Same comment as above, I think you can remove this, we'll probably solve it in another way, in another issue.